neurips2022/track2/train/utility_IL.py

```diff
@@ -64,7 +64,7 @@ def load_data_scratch(dataset_path, save_path):
 
         for id in vehicle_ids:
             with open(
-                os.path.join(path, scenario) + "/Agent-history-vehicle-" + id + ".pkl",
+                os.path.join(path, scenario) + "/sumo_sumo_1_Agent-history-vehicle-" + id + ".pkl",
```

Comment:
If you hard code "sumo_sumo_*" string, what happens when a differently named hidden dataset is used during docker-based re-training?